src/app/static/src/app/store/modelRun/modelRun.ts

```diff
@@ -13,8 +13,6 @@ export interface ModelRunState extends ReadyState {
     result: ModelResultResponse | null
 }
 
-export const localStorageKey = "modelRun";
```

Comment:
Just noticed this superfluous const.

Reply:
Tests are failing now, looks like mutations is looking for that const.

Reply:
Sorry, should have waited for build to pass before submitting. Passing now.